### PR TITLE
[issue 22] categorize volunteer_type.names using first dash as delineation

### PR DESCRIPTION
**Pull request by ironunicorn**
I spoke with Harriet and she's fine with the style. Assuming the change is approved, I'll go ahead and add all the extra categories from the google form to nationbuilder once the code is live.

### CODE REVIEW

File: themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/pages_show_volunteer_signup.html [hunk 1]
```diff
@@ -48,16 +48,43 @@ <h4>Will you volunteer?</h4>
...
+        {% if category != " " %}
+        <div class="padtop">
+          {{ category }}
         </div>
-        {% endfor %}
+          {% for volunteer_type in page.volunteer_signup.volunteer_types %}
+            {% if volunteer_type.name contains category %}
+          <div class="row-fluid">
+            <div class="span12"><label class="checkbox">
+              {% assign type_info = volunteer_type.name | split: " - " %}
```
Comment (chrisflesher) [inline themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/pages_show_volunteer_signup.html:76]: Nit: What about `type_info` -> `volunteer_category`?

Comment (chrisflesher) [reply themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/pages_show_volunteer_signup.html:76]: Nit just means a suggested change but can ignore it if you want. https://google.github.io/eng-practices/review/reviewer/standard.html

File: themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/pages_show_volunteer_signup.html [hunk 2]
```diff
@@ -48,16 +48,43 @@ <h4>Will you volunteer?</h4>
...
+              {% assign type_name_to_remove = type_info[0] | append: " - " %}
```
Comment (chrisflesher) [inline themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/pages_show_volunteer_signup.html:77]: `volunteer_category_to_remove`?

**Review by chrisflesher — APPROVED** [review]
Pretty cool! Looks good to me.

File: themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/pages_show_volunteer_signup.html (hunk 1 above)
Comment (ironunicorn) [reply themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/pages_show_volunteer_signup.html:76]: Oh! Yes, both your naming recommendations sound much more descriptive. I'll make the changes, thanks!

**Comment by ironunicorn** [comment]
Okay! I've made the changes. I don't have write access to this repo so I can't merge with master myself, but once the file is merged I can make it live on nationbuilder.

**Review by chrisflesher — APPROVED** [review]
Looks good.

**Pull request merged**